tests/integration/src/utils.rs

```diff
@@ -38,6 +38,7 @@ use upgrader_api::{
 use uuid::Uuid;
 
 pub const ADMIN_GROUP_ID: Uuid = Uuid::from_u128(302240678275694148452352); // very first uuidv4 generated
+pub const OPERATOR_GROUP_ID: Uuid = Uuid::from_u128(302240678275694148452353); // very first uuidv4 generated
```

Comment:
```suggestion
pub const OPERATOR_GROUP_ID: Uuid = ADMIN_GROUP_ID; // very first uuidv4 generated
```
if possible

Reply:
Here's the comment that is wrong, the operator group id is the 2nd UUID.